js/views/inbox_view.js

```diff
@@ -107,11 +107,6 @@
 
       const inboxCollection = getInboxCollection();
 
-      this.listenTo(inboxCollection, 'messageError', () => {
-        if (this.networkStatusView) {
```

Comment:
I wasn't quite clear what this did? But since the networkStatusView is already rendered this code is redundant.

Reply:
It's because it likely needs to be re-rendered because state has changed.